### PR TITLE
NDS-86 Set up Gatsby documentation site

### PR DESCRIPTION
**Pull request by greypiglet**
https://nicedigital.atlassian.net/browse/NDS-86

### CODE REVIEW

File: components/nds-tabs/src/Tabs.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { slugify } from "@nice-digital/nds-core/es/utils";
+import { slugify } from "@nice-digital/nds-core/src/utils";
```
Comment (ediblecode) [inline components/nds-tabs/src/Tabs.js]: This should be es/lib not src shouldn't it as we don't include the src when we package and even if we did, webpack/babel wouldn't transiple a file from within node_modules by default?

File: lerna.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "packages": [
+    "components/*",
```
Comment (ediblecode) [inline lerna.json]: Nice

File: packages/docs/LICENSE [hunk 3]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 gatsbyjs
```
Comment (ediblecode) [inline packages/docs/LICENSE]: Can you change this to the same copyright name as the root?

File: packages/docs/package.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "docs",
```
Comment (ediblecode) [inline packages/docs/package.json]: Maybe change this to @nice-digital/design-system-docs or something?

File: packages/docs/package.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
```
Comment (ediblecode) [inline packages/docs/package.json]: Can you update the description and author?

File: packages/docs/package.json [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsbyjs/gatsby-starter-default"
```
Comment (ediblecode) [inline packages/docs/package.json]: Can you update repo/bugs URL too please?

File: packages/docs/package.json [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    "gatsby"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
```
Comment (ediblecode) [inline packages/docs/package.json]: I think we'll use jest from the root in which case can you change this echo to say run tests from the root or something?

File: packages/docs/src/components/layout.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Header, Footer } from "@nice-digital/global-nav";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import "./main.scss";
+
+interface ILayout {
+	children: React.ReactNode;
+}
+
+const Layout = (props: ILayout): React.ReactElement => (
```
Comment (ediblecode) [inline packages/docs/src/components/layout.tsx]: Can you write a simple test for a component/page within the gatsby site just to prove that running tests from the root picks up files in here?

File: components/nds-tabs/src/Tabs.js (hunk 1 above)
Comment (greypiglet) [reply components/nds-tabs/src/Tabs.js]: absolutely, i think webstorm dynamically changed this at some point, i hadn't manually edited it!

File: packages/docs/src/components/layout.tsx (hunk 8 above)
Comment (greypiglet) [reply packages/docs/src/components/layout.tsx]: @ediblecode Is just a `todo` test okay for now?

File: components/nds-tabs/src/Tabs.js (hunk 1 above)
Comment (ediblecode) [reply components/nds-tabs/src/Tabs.js]: This is still src not es!

**Pull request merged**